releasenotes/notes/fix-signing-keys-dependency-84fb4c017cab72af.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    The weak dependency of datadog-agent, datadog-iot-agent and dogstatsd deb
```

Comment:
Should we modify the release note from #11447 instead? That change hasn't been released yet AFAICT

Reply:
It will be released in 7.35.0, so I guess it would make sense to differentiate it by adding a new note for 7.36.0.